src/Sulu/Component/Content/Import/Format/Xliff12.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Content\Import\Format;
+
+use Sulu\Component\Content\Import\WebspaceFormatImportInterface;
+use Symfony\Component\Config\Util\XmlUtils;
+
+/**
+ * Import an xliff 1.2 file.
+ */
+class Xliff12 implements WebspaceFormatImportInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function parse($filePath, $locale)
+    {
+        $dom = XmlUtils::loadFile($filePath);
+
+        return $this->extractData($dom, $locale);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getProperty($name, $data, $contentTypeName = null, $extension = null, $default = null)
+    {
+        $propertyName = '';
+
+        if ($extension) {
+            $propertyName = $extension . '-';
+        }
+
+        $propertyName .= $name;
+
+        if (!isset($data[$propertyName])) {
+            return $default;
+        }
+
+        $property = $data[$propertyName];
+        $property['type'] = $contentTypeName;
+
+        return $property;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getPropertyData($name, $data, $contentTypeName = null, $extension = null, $default = null)
+    {
+        $property = $this->getProperty($name, $data, $contentTypeName, $extension);
+
+        if (isset($property['children'])) {
+            $data = [];
+            foreach ($property['children'] as $key => $child) {
+                $childProperties = [];
+                foreach (array_keys($child) as $childKey) {
```

Comment:
use early return and extract the second foreach to an own method to avoid nesting loops